troposphere/static/css/app/quota.scss

```diff
@@ -0,0 +1,40 @@
+textarea, select{
+	width: 95%;
+}
+
+.quota-table{
+	float: left;
+	width: 50%;
+}
+
+.quota-detail{
+	width: 48%;
+	float: right;
+	position: relative;
+	border: 5px solid #007b89;
+}
+
+.quota-detail button{
+	margin: 2%;
+}
+
+.quota-detail div{
+	margin: 2%;
+}
+
+.quota-row, .quota-inline{
+	width: 100%;
+}
+
+.user-name a.active{
+	color: #007b89;
```

Comment:
Is this a custom color, or something that's part of the branding?  Does it map to one of the SCSS variables used for branding?